packages/styled-components/src/test/constants.test.js

```diff
@@ -87,6 +88,12 @@ describe('constants', () => {
       renderAndExpect(false, '');
     });
 
+    it('should be false in development NODE_ENV when window.SC_DISABLE_SPEEDY is set to false', () => {
+      process.env.NODE_ENV = 'development';
+      window.SC_DISABLE_SPEEDY = false;
+      renderAndExpect(false, '');
```

Comment:
Hey 👋, was wondering what `renderAndExpect` does? Many thanks, this is just out of curiosity 😄

Reply:
https://github.com/styled-components/styled-components/pull/3289/files#diff-8ee68021d59f52aef5b5d6fbd8a082e4R12-R28